content/appendix/audit_reports.md

```diff
@@ -33,6 +33,14 @@ This audit covers the implementation of Filecoin's builtin Actors, focusing on t
 
 ## Proofs
 
+### `2021-05-31` SnarkPack audit
+
+Two externals audits have been contracted on the cryptographic part of [SnarkPack](https://eprint.iacr.org/2021/529.pdf), that is used in the [FIP0009](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0009.md):
+- [Report](https://drive.google.com/file/d/1avwxOO6CK_nkX7AVV4P9l6XawNco3-m_/view) from [NCC group](https://www.nccgroup.com/us/)
```

Comment:
here's the final published link: https://research.nccgroup.com/wp-content/uploads/2021/06/NCC_Group_ProtocolLabs_FilecoinGroth16_Report_2021-06-02.pdf